core/src/main/scala/kafka/message/MessageAndOffset.scala

```diff
@@ -24,5 +24,14 @@ case class MessageAndOffset(message: Message, offset: Long) {
    * Compute the offset of the next message in the log
    */
   def nextOffset: Long = offset + 1
+
+  /**
+   * If this is a compressed message, get the firxst offset of the inner messages.
```

Comment:
typo firxst

Also, the next line of the comment seems to talk about the same thing.